Core/include/Acts/Surfaces/AnnulusBounds.hpp

```diff
@@ -209,6 +209,11 @@ class AnnulusBounds : public DiscBounds {
 
   /// Private helper method
   double squaredNorm(const Vector2& v, const SquareMatrix2& weight) const;
+
+private:
+  bool inside(const Vector2& lposition, double tolR,
+              double tolPhiR,
+              double scale) const;
```

Comment:
I think this can be replaced by a proper implementation of the inside check given a tolerance

Reply:
passing the "scale" parameter avoids a division. But anyway the way it is implemented duplicates code. So, as mentioned at a different place. This was the easiest way to implement it and was anyway more meant to be a proof of concept. And so far we only see that the concept does not work all that great ;-)